modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro

```diff
@@ -115,6 +115,20 @@ definition {
 			locator1 = "Header#H1_TITLE");
 	}
 
+	macro deleteClaim {
+		Button.click(button = "Delete");
+
+		Confirm(value1 = "Are you sure?");
+
+		PRMNavigator.openSitePage(pageName = "MDF Claims");
+
```

Comment:
Also @brunadmelo , could you separate the first two actions on the macro to make more generic and possible to delete a claim also with this? By separating the Button.click and Confirm is it possible to make more generic.